rs/tests/consensus/subnet_recovery/common.rs

```diff
@@ -90,15 +91,20 @@ pub const CHAIN_KEY_SUBNET_RECOVERY_TIMEOUT: Duration = Duration::from_secs(15 *
 /// an app subnet with the given number of nodes
 pub fn setup(
     nns_nodes: usize,
+    source_nodes: usize,
     app_nodes: usize,
     unassigned_nodes: usize,
     dkg_interval: u64,
     env: TestEnv,
 ) {
-    let mut nns = Subnet::new(SubnetType::System)
+    let nns = Subnet::new(SubnetType::System)
```

Comment:
NIT: I think the creation of the subnets can be done directly in the `.add_subnet` clause. This is how we do it in most tests, I'm not entirely sure why we don't do it here